File: opcua-webservice-spring/core/src/main/java/org/opcfoundation/webserver/service/message/digitaltwin/GetPropertySubElementsResponse.java
package org.opcfoundation.webserver.service.message.digitaltwin;

import java.util.HashSet;
import java.util.Set;

public class GetPropertySubElementsResponse {
    private final Set<String> subElementNames;

    public GetPropertySubElementsResponse()
    {
        this.subElementNames = new HashSet<>();
    }

    public void add(String subElementName)
    {
        subElementNames.add(subElementName);
    }

    public Set<String> getSubElementNames() {
        return subElementNames;
    }
}

File: opcua-webservice-node/source/src/server/service/message/digitialtwin/GetPropertySubElementsResponse.ts
export class GetPropertySubElementsResponse {
    private readonly _subElementNames: Set<string>;

    constructor() {
        this._subElementNames = new Set();
    }

    add(subElementName: string): void {
        this._subElementNames.add(subElementName);
    }

    get subElementNames(): Set<string> {
        return this._subElementNames;
    }
}